packages/gcp-pubsub/README.md

```
@@ -1490,6 +1490,92 @@ When a message cannot be processed (invalid format, schema validation failure, h
+          baseRetryDelayMs: 1000,  // Base delay for exponential backoff (default: 1000ms)
+          maxRetryDelayMs: 30000,  // Maximum delay between retries (default: 30000ms)
+        },
+      },
+      executionContext,
+    )
+  }
+}
+```
+
+**Exponential Backoff Formula:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)